### ISSUE TITLE
A commit has replaced all accent to a weid char `C)`

### ISSUE THREAD

**Issue by Kmaschta** [marmelab/sedy#36]
See this commit https://github.com/marmelab/marmelab.github.io/pull/156/commits/1887f6219cd0629552ce4eea01d80410f9513917

<details>
<summary>

Webhook payload</summary>

`````` json
{
  "action": "created",
  "comment": {
    "url": "https://api.github.com/repos/marmelab/marmelab.github.io/pulls/comments/77782663",
    "id": 77782663,
    "diff_hunk": "@@ -0,0 +1,214 @@\n+---\n+layout: post\n+title: \"Puissance 4 multi-joueur : retour d'expérience\"\n+excerpt: \"Première semaine parmi les équipes marmelab, premier challenge de la période d'intégration, 4 jours pour mettre au point un serveur de jeu puissance 4.\"\n+image: /images/blog/connect-four-small.png\n+author: <a href=\"https://github.com/floo51\">Florian</a>\n+tags:\n+- PHP\n+- symfony\n+- doctrine\n+---\n+\n+Première semaine parmi les équipes marmelab, premier challenge de la période d'intégration, 4 jours pour mettre au point un serveur de jeu puissance 4.\n+\n+Le sujet : 2 joueurs se connectent sur un site web et se rejoignent pour une partie de puissance 4. A tour de rôle, ils déposent un pion jusqu'à ce que l'un des deux gagne ou que le plateau soit rempli. Les technos : PHP7, Symfony 3.\n+\n+<img src=\"/images/blog/connect-four-big.png\" class=\"responsive\" />\n+\n+Les objectifs ? Se familiariser avec les méthodes et les technologies marmelab en soumettant son travail à l'avis de ses pairs.\n+\n+Qu'en ai-je retenu...\n+\n+## L'art de la sémantique\n+\n+> _[Définition](https://fr.wikipedia.org/wiki/S%C3%A9mantique):\n+La sémantique est une branche de la linguistique qui étudie les signifiés, ce dont on parle, ce que l'on veut énoncer. Sa branche symétrique, la syntaxe, concerne pour sa part le signifiant, sa forme, sa langue, sa graphie, sa grammaire, etc ; c'est la forme de l'énoncé._\n+\n+Ou pour faire plus simple et remettre en contexte, la sémantique est le fait de trouver le meilleur mot permettant de décrire un concept avec aussi peu d'ambiguïté que possible.\n+\n+Utiliser le langage le plus naturel possible et des termes au plus proche du réel permet de comprendre les liens entres nos objets et les concepts fonctionnels qu'ils nous servent à manipuler. Les commentaires deviennent alors superflux et le code se documente par lui-même.\n+\n+```diff\n+class Board\n+{\n+    const COLUMNS = 7;\n+    const ROWS = 6;\n+\n++   private $cells;\n+-   private $discs;\n+\n+    public function __construct()\n+    {\n+        $this->initializeCells();\n+    }\n+```\n+\n+Dans le modèle initial, le plateau de jeu contenait un tableau de pions (`discs`) puis, suite à un refactoring, une matrice de cellule; chaque cellule pouvant contenir un pion ou être vide.\n+\n+Lors d'un tel refactoring remettant en cause la signification d'un modèle de données, il est essentiel d'également considérer un refactoring sémantique.\n+Le nom `discs` n'est plus du tout adapté à ce que le concept représente, il pourrait donc être mal interprété par un futur développeur et amener à compromettre la maintenabilité générale du projet.\n+\n+## Best practices\n+\n+### Conventions & consistence",
    "path": "blog/_posts/2016-09-02-puissance-4-retour-experience.md",
    "position": 54,
    "original_position": 54,
    "commit_id": "44480e191b2f8fea904a17021f204a87323b3552",
    "original_commit_id": "44480e191b2f8fea904a17021f204a87323b3552",
    "user": {
      "login": "fzaninotto",
      "id": 99944,
      "avatar_url": "https://avatars.githubusercontent.com/u/99944?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/fzaninotto",
      "html_url": "https://github.com/fzaninotto",
      "followers_url": "https://api.github.com/users/fzaninotto/followers",
      "following_url": "https://api.github.com/users/fzaninotto/following{/other_user}",
      "gists_url": "https://api.github.com/users/fzaninotto/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/fzaninotto/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/fzaninotto/subscriptions",
      "organizations_url": "https://api.github.com/users/fzaninotto/orgs",
      "repos_url": "https://api.github.com/users/fzaninotto/repos",
      "events_url": "https://api.github.com/users/fzaninotto/events{/privacy}",
      "received_events_url": "https://api.github.com/users/fzaninotto/received_events",
      "type": "User",
      "site_admin": false
    },
    "body": "s/consistence/cohérence/",
    "created_at": "2016-09-07T08:50:56Z",
    "updated_at": "2016-09-07T08:50:56Z",
    "html_url": "https://github.com/marmelab/marmelab.github.io/pull/156#discussion_r77782663",
    "pull_request_url": "https://api.github.com/repos/marmelab/marmelab.github.io/pulls/156",
    "_links": {
      "self": {
        "href": "https://api.github.com/repos/marmelab/marmelab.github.io/pulls/comments/77782663"
      },
      "html": {
        "href": "https://github.com/marmelab/marmelab.github.io/pull/156#discussion_r77782663"
      },
      "pull_request": {
        "href": "https://api.github.com/repos/marmelab/marmelab.github.io/pulls/156"
      }
    }
  },
  "pull_request": {
    "url": "https://api.github.com/repos/marmelab/marmelab.github.io/pulls/156",
    "id": 83807210,
    "html_url": "https://github.com/marmelab/marmelab.github.io/pull/156",
    "diff_url": "https://github.com/marmelab/marmelab.github.io/pull/156.diff",
    "patch_url": "https://github.com/marmelab/marmelab.github.io/pull/156.patch",
    "issue_url": "https://api.github.com/repos/marmelab/marmelab.github.io/issues/156",
    "number": 156,
    "state": "open",
    "locked": false,
    "title": "[RFR] Add connect four article",
    "user": {
      "login": "floo51",
      "id": 2562270,
      "avatar_url": "https://avatars.githubusercontent.com/u/2562270?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/floo51",
      "html_url": "https://github.com/floo51",
      "followers_url": "https://api.github.com/users/floo51/followers",
      "following_url": "https://api.github.com/users/floo51/following{/other_user}",
      "gists_url": "https://api.github.com/users/floo51/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/floo51/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/floo51/subscriptions",
      "organizations_url": "https://api.github.com/users/floo51/orgs",
      "repos_url": "https://api.github.com/users/floo51/repos",
      "events_url": "https://api.github.com/users/floo51/events{/privacy}",
      "received_events_url": "https://api.github.com/users/floo51/received_events",
      "type": "User",
      "site_admin": false
    },
    "body": "- [x] Header\r\n- [x] Outline\r\n- [x] §1 : L'art du nommage\r\n- [x] §2 : Best practices\r\n- [x] §3 : Temps réel\r\n- [x] §4 : Conclusion",
    "created_at": "2016-09-02T14:05:16Z",
    "updated_at": "2016-09-07T08:50:56Z",
    "closed_at": null,
    "merged_at": null,
    "merge_commit_sha": "af4433e7169f9ba68de2354ce4450990bd4ed91e",
    "assignee": null,
    "assignees": [

    ],
    "milestone": null,
    "commits_url": "https://api.github.com/repos/marmelab/marmelab.github.io/pulls/156/commits",
    "review_comments_url": "https://api.github.com/repos/marmelab/marmelab.github.io/pulls/156/comments",
    "review_comment_url": "https://api.github.com/repos/marmelab/marmelab.github.io/pulls/comments{/number}",
    "comments_url": "https://api.github.com/repos/marmelab/marmelab.github.io/issues/156/comments",
    "statuses_url": "https://api.github.com/repos/marmelab/marmelab.github.io/statuses/44480e191b2f8fea904a17021f204a87323b3552",
    "head": {
      "label": "marmelab:connect_four",
      "ref": "connect_four",
      "sha": "44480e191b2f8fea904a17021f204a87323b3552",
      "user": {
        "login": "marmelab",
        "id": 3116319,
        "avatar_url": "https://avatars.githubusercontent.com/u/3116319?v=3",
        "gravatar_id": "",
        "url": "https://api.github.com/users/marmelab",
        "html_url": "https://github.com/marmelab",
        "followers_url": "https://api.github.com/users/marmelab/followers",
        "following_url": "https://api.github.com/users/marmelab/following{/other_user}",
        "gists_url": "https://api.github.com/users/marmelab/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/marmelab/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/marmelab/subscriptions",
        "organizations_url": "https://api.github.com/users/marmelab/orgs",
        "repos_url": "https://api.github.com/users/marmelab/repos",
        "events_url": "https://api.github.com/users/marmelab/events{/privacy}",
        "received_events_url": "https://api.github.com/users/marmelab/received_events",
        "type": "Organization",
        "site_admin": false
      },
      "repo": {
        "id": 13563896,
        "name": "marmelab.github.io",
        "full_name": "marmelab/marmelab.github.io",
        "owner": {
          "login": "marmelab",
          "id": 3116319,
          "avatar_url": "https://avatars.githubusercontent.com/u/3116319?v=3",
          "gravatar_id": "",
          "url": "https://api.github.com/users/marmelab",
          "html_url": "https://github.com/marmelab",
          "followers_url": "https://api.github.com/users/marmelab/followers",
          "following_url": "https://api.github.com/users/marmelab/following{/other_user}",
          "gists_url": "https://api.github.com/users/marmelab/gists{/gist_id}",
          "starred_url": "https://api.github.com/users/marmelab/starred{/owner}{/repo}",
          "subscriptions_url": "https://api.github.com/users/marmelab/subscriptions",
          "organizations_url": "https://api.github.com/users/marmelab/orgs",
          "repos_url": "https://api.github.com/users/marmelab/repos",
          "events_url": "https://api.github.com/users/marmelab/events{/privacy}",
          "received_events_url": "https://api.github.com/users/marmelab/received_events",
          "type": "Organization",
          "site_admin": false
        },
        "private": true,
        "html_url": "https://github.com/marmelab/marmelab.github.io",
        "description": "marmelab.com website",
        "fork": false,
        "url": "https://api.github.com/repos/marmelab/marmelab.github.io",
        "forks_url": "https://api.github.com/repos/marmelab/marmelab.github.io/forks",
        "keys_url": "https://api.github.com/repos/marmelab/marmelab.github.io/keys{/key_id}",
        "collaborators_url": "https://api.github.com/repos/marmelab/marmelab.github.io/collaborators{/collaborator}",
        "teams_url": "https://api.github.com/repos/marmelab/marmelab.github.io/teams",
        "hooks_url": "https://api.github.com/repos/marmelab/marmelab.github.io/hooks",
        "issue_events_url": "https://api.github.com/repos/marmelab/marmelab.github.io/issues/events{/number}",
        "events_url": "https://api.github.com/repos/marmelab/marmelab.github.io/events",
        "assignees_url": "https://api.github.com/repos/marmelab/marmelab.github.io/assignees{/user}",
        "branches_url": "https://api.github.com/repos/marmelab/marmelab.github.io/branches{/branch}",
        "tags_url": "https://api.github.com/repos/marmelab/marmelab.github.io/tags",
        "blobs_url": "https://api.github.com/repos/marmelab/marmelab.github.io/git/blobs{/sha}",
        "git_tags_url": "https://api.github.com/repos/marmelab/marmelab.github.io/git/tags{/sha}",
        "git_refs_url": "https://api.github.com/repos/marmelab/marmelab.github.io/git/refs{/sha}",
        "trees_url": "https://api.github.com/repos/marmelab/marmelab.github.io/git/trees{/sha}",
        "statuses_url": "https://api.github.com/repos/marmelab/marmelab.github.io/statuses/{sha}",
        "languages_url": "https://api.github.com/repos/marmelab/marmelab.github.io/languages",
        "stargazers_url": "https://api.github.com/repos/marmelab/marmelab.github.io/stargazers",
        "contributors_url": "https://api.github.com/repos/marmelab/marmelab.github.io/contributors",
        "subscribers_url": "https://api.github.com/repos/marmelab/marmelab.github.io/subscribers",
        "subscription_url": "https://api.github.com/repos/marmelab/marmelab.github.io/subscription",
        "commits_url": "https://api.github.com/repos/marmelab/marmelab.github.io/commits{/sha}",
        "git_commits_url": "https://api.github.com/repos/marmelab/marmelab.github.io/git/commits{/sha}",
        "comments_url": "https://api.github.com/repos/marmelab/marmelab.github.io/comments{/number}",
        "issue_comment_url": "https://api.github.com/repos/marmelab/marmelab.github.io/issues/comments{/number}",
        "contents_url": "https://api.github.com/repos/marmelab/marmelab.github.io/contents/{+path}",
        "compare_url": "https://api.github.com/repos/marmelab/marmelab.github.io/compare/{base}...{head}",
        "merges_url": "https://api.github.com/repos/marmelab/marmelab.github.io/merges",
        "archive_url": "https://api.github.com/repos/marmelab/marmelab.github.io/{archive_format}{/ref}",
        "downloads_url": "https://api.github.com/repos/marmelab/marmelab.github.io/downloads",
        "issues_url": "https://api.github.com/repos/marmelab/marmelab.github.io/issues{/number}",
        "pulls_url": "https://api.github.com/repos/marmelab/marmelab.github.io/pulls{/number}",
        "milestones_url": "https://api.github.com/repos/marmelab/marmelab.github.io/milestones{/number}",
        "notifications_url": "https://api.github.com/repos/marmelab/marmelab.github.io/notifications{?since,all,participating}",
        "labels_url": "https://api.github.com/repos/marmelab/marmelab.github.io/labels{/name}",
        "releases_url": "https://api.github.com/repos/marmelab/marmelab.github.io/releases{/id}",
        "deployments_url": "https://api.github.com/repos/marmelab/marmelab.github.io/deployments",
        "created_at": "2013-10-14T14:41:40Z",
        "updated_at": "2016-03-09T08:16:34Z",
        "pushed_at": "2016-09-07T08:45:28Z",
        "git_url": "git://github.com/marmelab/marmelab.github.io.git",
        "ssh_url": "git@github.com:marmelab/marmelab.github.io.git",
        "clone_url": "https://github.com/marmelab/marmelab.github.io.git",
        "svn_url": "https://github.com/marmelab/marmelab.github.io",
        "homepage": "marmelab.com",
        "size": 63165,
        "stargazers_count": 2,
        "watchers_count": 2,
        "language": "HTML",
        "has_issues": true,
        "has_downloads": true,
        "has_wiki": true,
        "has_pages": true,
        "forks_count": 0,
        "mirror_url": null,
        "open_issues_count": 8,
        "forks": 0,
        "open_issues": 8,
        "watchers": 2,
        "default_branch": "master"
      }
    },
    "base": {
      "label": "marmelab:master",
      "ref": "master",
      "sha": "a381505e838eece479fb149dc09860d7d9a1e527",
      "user": {
        "login": "marmelab",
        "id": 3116319,
        "avatar_url": "https://avatars.githubusercontent.com/u/3116319?v=3",
        "gravatar_id": "",
        "url": "https://api.github.com/users/marmelab",
        "html_url": "https://github.com/marmelab",
        "followers_url": "https://api.github.com/users/marmelab/followers",
        "following_url": "https://api.github.com/users/marmelab/following{/other_user}",
        "gists_url": "https://api.github.com/users/marmelab/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/marmelab/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/marmelab/subscriptions",
        "organizations_url": "https://api.github.com/users/marmelab/orgs",
        "repos_url": "https://api.github.com/users/marmelab/repos",
        "events_url": "https://api.github.com/users/marmelab/events{/privacy}",
        "received_events_url": "https://api.github.com/users/marmelab/received_events",
        "type": "Organization",
        "site_admin": false
      },
      "repo": {
        "id": 13563896,
        "name": "marmelab.github.io",
        "full_name": "marmelab/marmelab.github.io",
        "owner": {
          "login": "marmelab",
          "id": 3116319,
          "avatar_url": "https://avatars.githubusercontent.com/u/3116319?v=3",
          "gravatar_id": "",
          "url": "https://api.github.com/users/marmelab",
          "html_url": "https://github.com/marmelab",
          "followers_url": "https://api.github.com/users/marmelab/followers",
          "following_url": "https://api.github.com/users/marmelab/following{/other_user}",
          "gists_url": "https://api.github.com/users/marmelab/gists{/gist_id}",
          "starred_url": "https://api.github.com/users/marmelab/starred{/owner}{/repo}",
          "subscriptions_url": "https://api.github.com/users/marmelab/subscriptions",
          "organizations_url": "https://api.github.com/users/marmelab/orgs",
          "repos_url": "https://api.github.com/users/marmelab/repos",
          "events_url": "https://api.github.com/users/marmelab/events{/privacy}",
          "received_events_url": "https://api.github.com/users/marmelab/received_events",
          "type": "Organization",
          "site_admin": false
        },
        "private": true,
        "html_url": "https://github.com/marmelab/marmelab.github.io",
        "description": "marmelab.com website",
        "fork": false,
        "url": "https://api.github.com/repos/marmelab/marmelab.github.io",
        "forks_url": "https://api.github.com/repos/marmelab/marmelab.github.io/forks",
        "keys_url": "https://api.github.com/repos/marmelab/marmelab.github.io/keys{/key_id}",
        "collaborators_url": "https://api.github.com/repos/marmelab/marmelab.github.io/collaborators{/collaborator}",
        "teams_url": "https://api.github.com/repos/marmelab/marmelab.github.io/teams",
        "hooks_url": "https://api.github.com/repos/marmelab/marmelab.github.io/hooks",
        "issue_events_url": "https://api.github.com/repos/marmelab/marmelab.github.io/issues/events{/number}",
        "events_url": "https://api.github.com/repos/marmelab/marmelab.github.io/events",
        "assignees_url": "https://api.github.com/repos/marmelab/marmelab.github.io/assignees{/user}",
        "branches_url": "https://api.github.com/repos/marmelab/marmelab.github.io/branches{/branch}",
        "tags_url": "https://api.github.com/repos/marmelab/marmelab.github.io/tags",
        "blobs_url": "https://api.github.com/repos/marmelab/marmelab.github.io/git/blobs{/sha}",
        "git_tags_url": "https://api.github.com/repos/marmelab/marmelab.github.io/git/tags{/sha}",
        "git_refs_url": "https://api.github.com/repos/marmelab/marmelab.github.io/git/refs{/sha}",
        "trees_url": "https://api.github.com/repos/marmelab/marmelab.github.io/git/trees{/sha}",
        "statuses_url": "https://api.github.com/repos/marmelab/marmelab.github.io/statuses/{sha}",
        "languages_url": "https://api.github.com/repos/marmelab/marmelab.github.io/languages",
        "stargazers_url": "https://api.github.com/repos/marmelab/marmelab.github.io/stargazers",
        "contributors_url": "https://api.github.com/repos/marmelab/marmelab.github.io/contributors",
        "subscribers_url": "https://api.github.com/repos/marmelab/marmelab.github.io/subscribers",
        "subscription_url": "https://api.github.com/repos/marmelab/marmelab.github.io/subscription",
        "commits_url": "https://api.github.com/repos/marmelab/marmelab.github.io/commits{/sha}",
        "git_commits_url": "https://api.github.com/repos/marmelab/marmelab.github.io/git/commits{/sha}",
        "comments_url": "https://api.github.com/repos/marmelab/marmelab.github.io/comments{/number}",
        "issue_comment_url": "https://api.github.com/repos/marmelab/marmelab.github.io/issues/comments{/number}",
        "contents_url": "https://api.github.com/repos/marmelab/marmelab.github.io/contents/{+path}",
        "compare_url": "https://api.github.com/repos/marmelab/marmelab.github.io/compare/{base}...{head}",
        "merges_url": "https://api.github.com/repos/marmelab/marmelab.github.io/merges",
        "archive_url": "https://api.github.com/repos/marmelab/marmelab.github.io/{archive_format}{/ref}",
        "downloads_url": "https://api.github.com/repos/marmelab/marmelab.github.io/downloads",
        "issues_url": "https://api.github.com/repos/marmelab/marmelab.github.io/issues{/number}",
        "pulls_url": "https://api.github.com/repos/marmelab/marmelab.github.io/pulls{/number}",
        "milestones_url": "https://api.github.com/repos/marmelab/marmelab.github.io/milestones{/number}",
        "notifications_url": "https://api.github.com/repos/marmelab/marmelab.github.io/notifications{?since,all,participating}",
        "labels_url": "https://api.github.com/repos/marmelab/marmelab.github.io/labels{/name}",
        "releases_url": "https://api.github.com/repos/marmelab/marmelab.github.io/releases{/id}",
        "deployments_url": "https://api.github.com/repos/marmelab/marmelab.github.io/deployments",
        "created_at": "2013-10-14T14:41:40Z",
        "updated_at": "2016-03-09T08:16:34Z",
        "pushed_at": "2016-09-07T08:45:28Z",
        "git_url": "git://github.com/marmelab/marmelab.github.io.git",
        "ssh_url": "git@github.com:marmelab/marmelab.github.io.git",
        "clone_url": "https://github.com/marmelab/marmelab.github.io.git",
        "svn_url": "https://github.com/marmelab/marmelab.github.io",
        "homepage": "marmelab.com",
        "size": 63165,
        "stargazers_count": 2,
        "watchers_count": 2,
        "language": "HTML",
        "has_issues": true,
        "has_downloads": true,
        "has_wiki": true,
        "has_pages": true,
        "forks_count": 0,
        "mirror_url": null,
        "open_issues_count": 8,
        "forks": 0,
        "open_issues": 8,
        "watchers": 2,
        "default_branch": "master"
      }
    },
    "_links": {
      "self": {
        "href": "https://api.github.com/repos/marmelab/marmelab.github.io/pulls/156"
      },
      "html": {
        "href": "https://github.com/marmelab/marmelab.github.io/pull/156"
      },
      "issue": {
        "href": "https://api.github.com/repos/marmelab/marmelab.github.io/issues/156"
      },
      "comments": {
        "href": "https://api.github.com/repos/marmelab/marmelab.github.io/issues/156/comments"
      },
      "review_comments": {
        "href": "https://api.github.com/repos/marmelab/marmelab.github.io/pulls/156/comments"
      },
      "review_comment": {
        "href": "https://api.github.com/repos/marmelab/marmelab.github.io/pulls/comments{/number}"
      },
      "commits": {
        "href": "https://api.github.com/repos/marmelab/marmelab.github.io/pulls/156/commits"
      },
      "statuses": {
        "href": "https://api.github.com/repos/marmelab/marmelab.github.io/statuses/44480e191b2f8fea904a17021f204a87323b3552"
      }
    }
  },
  "repository": {
    "id": 13563896,
    "name": "marmelab.github.io",
    "full_name": "marmelab/marmelab.github.io",
    "owner": {
      "login": "marmelab",
      "id": 3116319,
      "avatar_url": "https://avatars.githubusercontent.com/u/3116319?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/marmelab",
      "html_url": "https://github.com/marmelab",
      "followers_url": "https://api.github.com/users/marmelab/followers",
      "following_url": "https://api.github.com/users/marmelab/following{/other_user}",
      "gists_url": "https://api.github.com/users/marmelab/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/marmelab/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/marmelab/subscriptions",
      "organizations_url": "https://api.github.com/users/marmelab/orgs",
      "repos_url": "https://api.github.com/users/marmelab/repos",
      "events_url": "https://api.github.com/users/marmelab/events{/privacy}",
      "received_events_url": "https://api.github.com/users/marmelab/received_events",
      "type": "Organization",
      "site_admin": false
    },
    "private": true,
    "html_url": "https://github.com/marmelab/marmelab.github.io",
    "description": "marmelab.com website",
    "fork": false,
    "url": "https://api.github.com/repos/marmelab/marmelab.github.io",
    "forks_url": "https://api.github.com/repos/marmelab/marmelab.github.io/forks",
    "keys_url": "https://api.github.com/repos/marmelab/marmelab.github.io/keys{/key_id}",
    "collaborators_url": "https://api.github.com/repos/marmelab/marmelab.github.io/collaborators{/collaborator}",
    "teams_url": "https://api.github.com/repos/marmelab/marmelab.github.io/teams",
    "hooks_url": "https://api.github.com/repos/marmelab/marmelab.github.io/hooks",
    "issue_events_url": "https://api.github.com/repos/marmelab/marmelab.github.io/issues/events{/number}",
    "events_url": "https://api.github.com/repos/marmelab/marmelab.github.io/events",
    "assignees_url": "https://api.github.com/repos/marmelab/marmelab.github.io/assignees{/user}",
    "branches_url": "https://api.github.com/repos/marmelab/marmelab.github.io/branches{/branch}",
    "tags_url": "https://api.github.com/repos/marmelab/marmelab.github.io/tags",
    "blobs_url": "https://api.github.com/repos/marmelab/marmelab.github.io/git/blobs{/sha}",
    "git_tags_url": "https://api.github.com/repos/marmelab/marmelab.github.io/git/tags{/sha}",
    "git_refs_url": "https://api.github.com/repos/marmelab/marmelab.github.io/git/refs{/sha}",
    "trees_url": "https://api.github.com/repos/marmelab/marmelab.github.io/git/trees{/sha}",
    "statuses_url": "https://api.github.com/repos/marmelab/marmelab.github.io/statuses/{sha}",
    "languages_url": "https://api.github.com/repos/marmelab/marmelab.github.io/languages",
    "stargazers_url": "https://api.github.com/repos/marmelab/marmelab.github.io/stargazers",
    "contributors_url": "https://api.github.com/repos/marmelab/marmelab.github.io/contributors",
    "subscribers_url": "https://api.github.com/repos/marmelab/marmelab.github.io/subscribers",
    "subscription_url": "https://api.github.com/repos/marmelab/marmelab.github.io/subscription",
    "commits_url": "https://api.github.com/repos/marmelab/marmelab.github.io/commits{/sha}",
    "git_commits_url": "https://api.github.com/repos/marmelab/marmelab.github.io/git/commits{/sha}",
    "comments_url": "https://api.github.com/repos/marmelab/marmelab.github.io/comments{/number}",
    "issue_comment_url": "https://api.github.com/repos/marmelab/marmelab.github.io/issues/comments{/number}",
    "contents_url": "https://api.github.com/repos/marmelab/marmelab.github.io/contents/{+path}",
    "compare_url": "https://api.github.com/repos/marmelab/marmelab.github.io/compare/{base}...{head}",
    "merges_url": "https://api.github.com/repos/marmelab/marmelab.github.io/merges",
    "archive_url": "https://api.github.com/repos/marmelab/marmelab.github.io/{archive_format}{/ref}",
    "downloads_url": "https://api.github.com/repos/marmelab/marmelab.github.io/downloads",
    "issues_url": "https://api.github.com/repos/marmelab/marmelab.github.io/issues{/number}",
    "pulls_url": "https://api.github.com/repos/marmelab/marmelab.github.io/pulls{/number}",
    "milestones_url": "https://api.github.com/repos/marmelab/marmelab.github.io/milestones{/number}",
    "notifications_url": "https://api.github.com/repos/marmelab/marmelab.github.io/notifications{?since,all,participating}",
    "labels_url": "https://api.github.com/repos/marmelab/marmelab.github.io/labels{/name}",
    "releases_url": "https://api.github.com/repos/marmelab/marmelab.github.io/releases{/id}",
    "deployments_url": "https://api.github.com/repos/marmelab/marmelab.github.io/deployments",
    "created_at": "2013-10-14T14:41:40Z",
    "updated_at": "2016-03-09T08:16:34Z",
    "pushed_at": "2016-09-07T08:45:28Z",
    "git_url": "git://github.com/marmelab/marmelab.github.io.git",
    "ssh_url": "git@github.com:marmelab/marmelab.github.io.git",
    "clone_url": "https://github.com/marmelab/marmelab.github.io.git",
    "svn_url": "https://github.com/marmelab/marmelab.github.io",
    "homepage": "marmelab.com",
    "size": 63165,
    "stargazers_count": 2,
    "watchers_count": 2,
    "language": "HTML",
    "has_issues": true,
    "has_downloads": true,
    "has_wiki": true,
    "has_pages": true,
    "forks_count": 0,
    "mirror_url": null,
    "open_issues_count": 8,
    "forks": 0,
    "open_issues": 8,
    "watchers": 2,
    "default_branch": "master"
  },
  "organization": {
    "login": "marmelab",
    "id": 3116319,
    "url": "https://api.github.com/orgs/marmelab",
    "repos_url": "https://api.github.com/orgs/marmelab/repos",
    "events_url": "https://api.github.com/orgs/marmelab/events",
    "hooks_url": "https://api.github.com/orgs/marmelab/hooks",
    "issues_url": "https://api.github.com/orgs/marmelab/issues",
    "members_url": "https://api.github.com/orgs/marmelab/members{/member}",
    "public_members_url": "https://api.github.com/orgs/marmelab/public_members{/member}",
    "avatar_url": "https://avatars.githubusercontent.com/u/3116319?v=3",
    "description": null
  },
  "sender": {
    "login": "fzaninotto",
    "id": 99944,
    "avatar_url": "https://avatars.githubusercontent.com/u/99944?v=3",
    "gravatar_id": "",
    "url": "https://api.github.com/users/fzaninotto",
    "html_url": "https://github.com/fzaninotto",
    "followers_url": "https://api.github.com/users/fzaninotto/followers",
    "following_url": "https://api.github.com/users/fzaninotto/following{/other_user}",
    "gists_url": "https://api.github.com/users/fzaninotto/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/fzaninotto/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/fzaninotto/subscriptions",
    "organizations_url": "https://api.github.com/users/fzaninotto/orgs",
    "repos_url": "https://api.github.com/users/fzaninotto/repos",
    "events_url": "https://api.github.com/users/fzaninotto/events{/privacy}",
    "received_events_url": "https://api.github.com/users/fzaninotto/received_events",
    "type": "User",
    "site_admin": false
  }
}
``````

</details>

**Comment by daj0ker** [comment]
Hi! I'm interested. The link appears to be broken though.

**Comment by Kmaschta** [comment]
Thank you @daj0ker!

The link goes to a private repository, that's why I pasted the payload in the issue.
Let me know if I can get you anything, like the resulting commit diff, etc.

**Comment by Kmaschta** [comment]
Fixed by af1310a4ab50404c3d9c3c3c56a2cebce16c8929
It'll be merged soon
